pkg/ccl/crosscluster/logical/logical_replication_job_test.go

```diff
@@ -509,6 +511,113 @@ func TestPreviouslyInterestingTables(t *testing.T) {
 	}
 }
 
+func TestMeasurePlanChange(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+
+	makeProc := func(dstID base.SQLInstanceID, srcIDs []base.SQLInstanceID) []physicalplan.Processor {
+		result := make([]physicalplan.Processor, 0, len(srcIDs))
+		for _, id := range srcIDs {
+			sp := execinfrapb.LogicalReplicationWriterSpec{}
+			sp.PartitionSpec = execinfrapb.StreamIngestionPartitionSpec{SrcInstanceID: id, DestInstanceID: dstID}
+			result = append(result, physicalplan.Processor{SQLInstanceID: dstID,
+				Spec: execinfrapb.ProcessorSpec{
+					Core: execinfrapb.ProcessorCoreUnion{
+						LogicalReplicationWriter: &sp}}})
+		}
+		return result
+	}
+
+	makePlan := func(procs ...[]physicalplan.Processor) sql.PhysicalPlan {
+		allProcs := make([]physicalplan.Processor, len(procs))
+
+		for _, list := range procs {
+			allProcs = append(allProcs, list...)
+		}
+
+		plan := sql.PhysicalPlan{}
+		plan.PhysicalInfrastructure = &physicalplan.PhysicalInfrastructure{Processors: allProcs}
+		return plan
+	}
+	for _, tc := range []struct {
+		name   string
+		before sql.PhysicalPlan
+		after  sql.PhysicalPlan
+		frac   float64
+	}{
+		{
+			name:   "same node",
```

Comment:
what coverage is this adding? this looks essentially identical to TestMeasurePlanChange in stream_ingestion_dist_test.go

Instead, i'd add a regression test case to the original test.

Reply:
I removed getNodes() from the test file so the test uses the real getNodes instead, which should cover any failures

Reply:
aha, LDR and PCR have different getNodes implementations. This looks good to merge once you add a regression test case.